### DISCUSSION TITLE
How to display something from db to top bar in filament(dynamic)?

### DISCUSSION THREAD

**Discussion by GiroeStefanBogdan** [filamentphp/filament#13030] [Help]
Hello. I want to display the number of credits(a column from db) in top bar in a filament application. 
If the column changes from db than the credits number I want to change as well without the need to refresh the page. 
I tried something like this and it worked but it was not dynamic: 

CompanyPanelProvider:
` ->renderHook(
                name: 'panels::topbar.start',
                hook: fn (): View => view('credits'),
            )`
            
            Here is the credits.blade.php: 
            
            
            
            
            
            
            credits.blade.php
            ```<?php

use App\Models\Business;
use App\Models\Subscription;

$business = Business::query()->where('name', 'GSD')->first();

if ($business) {
    $subscriptionId = $business['subscription_id'];
    $subscription = Subscription::find($subscriptionId);

    if($subscription){
        $credits = $subscription->credits;
    }
}else{
    $credits = 'undefined';
}

?>
<x-filament::badge size="md" color="warning">
    {{ $credits }} credits

</x-filament::badge>
```

**Comment by leandrocfe** [comment]
Maybe wire:poll? https://livewire.laravel.com/docs/wire-poll#basic-usage

**Reply by GiroeStefanBogdan** [reply]
Ok. Maybe wire:poll will work
But how can I call the livewire component from AdminPanelProvider ?

It is not working like this: 
`   ->renderHook(
                name: 'panels::topbar.start',
                hook: fn (): View => view('livewire.credits'),
            )`

**Comment by jiteshgolecha** [comment]
```
->renderHook(
                'panels::topbar.start',
                fn() => view('components.active-task')->with('activeTimesheet', TaskTimesheet::where('user_id', auth()->id())->whereNull('end')->first())
            )
```
